pom.xml

```diff
@@ -115,19 +115,9 @@ THE SOFTWARE.
       <version>1.3</version>
     </dependency>
     <dependency>
-      <groupId>${project.groupId}</groupId>
-      <artifactId>jenkins-test-harness-htmlunit</artifactId>
-      <version>2.18-1</version>
-      <exclusions>
-        <exclusion>
-          <groupId>org.eclipse.jetty</groupId>
-          <artifactId>jetty-io</artifactId>
-        </exclusion>
-        <exclusion>
-          <groupId>org.eclipse.jetty</groupId>
-          <artifactId>jetty-util</artifactId>
-        </exclusion>
-      </exclusions>
+      <groupId>net.sourceforge.htmlunit</groupId>
+      <artifactId>htmlunit</artifactId>
```

Comment:
No, I think we rather wanted the `jenkins-test-harness-htmlunit` to be updated to pull in the upstream 2.30 version, then cut a `2.30-1` release of that which would continue to do the shading.